### DISCUSSION TITLE
how to use new twitter icon?

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#215] [Q&A] [answered]
Twitter has updated its brand name and logo. Fontawesome has this as well.

<i class="fa-brands fa-x-twitter"></i>

But, it does not seem to be recognized for any icon that has a hyphen('-') in between. Any help will be appreciated.

**Comment by vincerubinetti** [comment, marked as answer]
Since this change was so recent, it's in a later version of Font Awesome. You'll need to upgrade it to `6.4.2` in `_includes/fonts.html`.

See this:
https://fontawesome.com/icons/x-twitter?f=brands&s=solid
